src/components/custom-field-input-text/custom-field-input-text.jsx

```diff
@@ -57,8 +69,10 @@ CustomFieldInputText.defaultProps = {
   helpText: undefined,
   id: undefined,
   name: undefined,
+  onChange: () => {},
   onClick: () => {},
   placeholder: undefined,
   required: false,
+  type: 'text',
```

Comment:
We might want to drop this if we're going to stick with `text`

Reply:
I think we probably still want to give users the option to customize this, especially if for whatever reason the text being entered is sensitive and needs `password`.